config/routes.yaml

```
@@ -2,4 +2,4 @@ controllers:
     resource:
         path: ../src/Controller/
         namespace: App\Controller
-    type: attribute
+    type: attribute
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)